CommandLine/emake-tests/Extensions/Json/short-json-converter-test.cpp

```diff
@@ -0,0 +1,404 @@
+#include "../../ENIGMAsystem/SHELL/Universal_System/Extensions/Json/short_json_converter.cpp"
+#include "../../ENIGMAsystem/SHELL/Universal_System/Extensions/Json/short_json_converter.h"
+
```

Comment:
I don't know what this is, but it's working that way, maybe I should use some guards?